app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -22,8 +22,9 @@
 		'location' => $location))
 	<div class="basic-info">
 		<div class="addresses">
-			<address>{{ $location->address }}</address>
-			<a href="{{ $location->getExternalWebURL() }}">{{ $location->getExternalWebURL() }}</a>
+		<p><em class="fa fa-map-marker"></em>{{ $location->address }}</p>
+		@if(!is_null($location->phone_number))<p><em class="fa fa-phone"></em>{{ $location->phone_number }}</p>@endif
```

Comment:
I like the phone number icon you added with fa-phone so keep that but wrap your em and the phone number text in an "a" element.

Wrapping with an "a" element can help us with search-engine optimization and potentially to help phone calling software recognize phone numbers on AccessLocator.

Also, prefix the href value with "tel:" to indicate that it is a link to a phone number.  

More detail on this type of link is at https://css-tricks.com/the-current-state-of-telephone-links/.